src/RecipeRepository.js

```diff
@@ -4,9 +4,18 @@ class RecipeRepository {
     this.ingredientIndex = allIngredients
   }
 
-  filterByTag (tag) {
-    return this.recipeIndex.filter(recipe => {
-      return recipe.tags.includes(tag)
+  filterByTag () {
+    const tagsInput = Object.values(arguments)
+    let tagStack = tagsInput.reduce((ret, tag) => {
+      if (tagsInput.indexOf(tag) === tagsInput.length - 1) {
+        ret += `'${tag}'`
+      } else {
+        ret += `'${tag}' && `
```

Comment:
Though this is strict additional, the either or operator would also be valid here